scripts/run_lighthouse_tests.py

```diff
@@ -96,12 +141,14 @@ def export_url(url):
         return
 
 
-def run_lighthouse_checks():
+def run_lighthouse_checks(is_performance_mode):
     """Runs the lighthouse checks through the .lighthouserc.js config."""
```

Comment:
Missing Args in docstring.

Also, I suggest declaring top-level constants LIGHTHOUSE_MODE_PERFORMANCE and LIGHTHOUSE_MODE_ACCESSIBILITY. Pass the constant instead. This is because it's not really clear that either of these is a sensible "default" value to be overridden (conceptually).

Reply:
1. Missing Args [Done]
2. LIGHTHOUSE_MODE_PERFORMANCE and LIGHTHOUSE_MODE_ACCESSIBILITY. [Done?]